zebra-utils/src/bin/zebra-tip-height/main.rs

```diff
@@ -0,0 +1,54 @@
+//! Prints the chain tip height stored in Zebra's state.
```

Comment:
Question: would this utility be better in zebra-state, so we can minimise its dependencies?

Reply:
I think we'll get the fastest build and easiest deployment by adding this utility as a sub-command of `zebrad` (like `zebrad copy-state`).

This also allows us to use whatever configuration `zebrad` has loaded, including the `-c` command-line argument.

Reply:
I'm not sure. I tried this now, and it seems that I've spent a lot more time changing things during the review of this PR than when I created the first implementation, and this seems like a sign of feature creep 😅 I was assuming this was a small utility to help with CI, so it doesn't have to be perfect.

Moving it to be a sub-command might be useful, but it breaks the tool because then it prints the logs from the state service, and I think we need a custom config to disable them (i.e., to set the log level to `warn` or something). I'm not sure what's the solution here, but I pushed a separate branch with the work as far as I got ([`tip-height-subcmd`](https://github.com/ZcashFoundation/zebra/compare/tip-height-subcmd)).

Let me know what you think and how we should proceed with this.

Reply:
> it seems that I've spent a lot more time changing things during the review of this PR than when I created the first implementation, and this seems like a sign of feature creep 😅 I was assuming this was a small utility to help with CI, so it doesn't have to be perfect.

Fair call, it might have been better to sort this out as part of a design discussion, rather than in the PR.

> Moving it to be a sub-command might be useful

It will be seamless in CI, because the zebrad binary is already available. So I think it will help a lot.

> but it breaks the tool because then it prints the logs from the state service, and I think we need a custom config to disable them (i.e., to set the log level to `warn` or something). I'm not sure what's the solution here, but I pushed a separate branch with the work as far as I got ([`tip-height-subcmd`](https://github.com/ZcashFoundation/zebra/compare/tip-height-subcmd)).

I can fix this, we can adjust the default verbosity for the tip command. Give me a few minutes.

Reply:
Awesome, thanks! Feel free to push that branch into this one if you want to 👍

Reply:
I pushed commit 324b537e254d4647151ed986d0302eea34a1e2d3 to ([tip-height-subcmd](https://github.com/ZcashFoundation/zebra/compare/tip-height-subcmd). It makes a bunch of commands quiet by default, including `tip-height`.

What do you think?

Reply:
I just pushed it over this branch

Reply:
Perfect! It looks great :D